### ISSUE TITLE
Accessibility Section icon

### ISSUE THREAD

**Issue by jgibsmith13** [LACMTA/ride-metro-net#23]
I think we should be using a more general Accessibility icon than one for elevators seen as this covers, elevators, escalators and could expand to cover more than those two things? (that pedestrian bridge closure for instance that references an ADA shuttle being available)

I guess this is the universal access symbol to include all types of disabilities rather than just the wheelchair.

<img width="181" height="148" alt="Image" src="https://github.com/user-attachments/assets/45f6bc93-d4b3-4daf-b256-63727f4e8448" />

**Comment by pswoodworth** [comment]
Yep makes sense!
